examples/cpp/BUILD-CPP-EXAMPLES.md

```diff
@@ -0,0 +1,28 @@
+# Geode Native C++ Examples
+
+## Prerequisites
+* Install [Apache Geode](https://geode.apache.org)
+* Install [CMake](https://cmake.org/download/)
+* Build and install [Apache Geode Native](https://github.com/apache/geode-native)
+* For Windows, Visual Studio 2015
+
+## Building the Examples
+
+1. Copy the `examples` directory from the native client installation folder to a folder in your workspace.
+
+1. Navigate to the directory for a specific example:
+
+    ```
+    $ cd workspace/examples/cpp/<example>
+    $ mkdir build
+    $ cd build
+    $ cmake ..
```

Comment:
This requires additional parameters for Windows, specifically the `-G"Visual Studio 14 2015 Win64" -Thost=x64` business we're all used to.